docs/design/interoperability/philosophy_and_goals.md

```diff
@@ -64,7 +64,7 @@ Other language interoperability layers that may offer useful examples are:
     for Carbon.
 
 -   [JavaScript/TypeScript](https://www.typescriptlang.org/docs/handbook/migrating-from-javascript.html)
-    is similar to C/C++, where one language is essentially a subset of the
+    is similar to C and C++, where one language is essentially a subset of the
```

Comment:
```suggestion
    is similar to C/C++, where one language is essentially a subset of the
```
Maybe revert this one, since the pattern here is to list "Base language/Language with interop" in a particular format. (If we want a change here, we should make it across all the language pairs.)